spec/requests/providers/means_summaries_spec.rb

```diff
@@ -31,10 +33,36 @@
       expect(response.body).to include(ActiveSupport::NumberHelper.number_to_currency(total_amount))
     end
 
+    it 'displays the correct vehicles details' do
+      expect(response.body).to include(number_to_currency(vehicle.estimated_value, unit: '£'))
+      expect(response.body).to include(number_to_currency(vehicle.payment_remaining, unit: '£'))
+      expect(response.body).to include(vehicle.purchased_on.to_s)
+      expect(response.body).to include(I18n.translate('shared.check_answers_vehicles.providers.heading'))
+      expect(response.body).to include(I18n.translate('shared.check_answers_vehicles.providers.own'))
```

Comment:
Really picky - but as the `t` alias is universally used for I18n's `translate` method in views etc, I think you can use it here too `I18n.t('shared.check_answers_vehicles.providers.own')` and elsewhere in this PR.